src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionToolOptions.cs

```diff
@@ -44,6 +44,7 @@ public sealed class RoslynInsertionToolOptions
             string componentBuildProjectName,
             string componentBuildQueueName,
             string componentBranchName,
+            string componentGitHubRepoName,
```

Comment:
thank you for adding the command line argument. extremely looking forward to turning this into a record.